src/smolagents/tools.py

```diff
@@ -232,7 +217,7 @@ def save(self, output_dir):
             tool_code = textwrap.dedent(
                 f"""
             from smolagents import Tool
-            from typing import Optional
+            from typing import Any, Optional
```

Comment:
I also found out that the way description is maintained here breaks if one has multiple lines.
Can't suggest an edit so created a separate PR for it: https://github.com/huggingface/smolagents/pull/613